lib/improver/nowcasting/optical_flow.py

```diff
@@ -319,6 +324,9 @@ def smooth(self, field, radius, method='box'):
         Smoothing method using a square ('box') or circular kernel.  Kernel
         smoothing with a radius of 1 has no effect.
 
+        Smoothing with the "box" argument is equivalent to the method
+        in equation 7 in Bowler et al. 2004.
```

Comment:
I think the reference is clear without the repeat in the function specific doc-string.